TestPlaybooks/playbook-TestHelloWorld.yml

```diff
@@ -0,0 +1,182 @@
+      - '5'
+    separatecontext: false
+    conditions:
+    - label: 'yes'
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: hello
+            iscontext: true
+          right:
+            value:
+              simple: Hello DBot
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 370\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '3':
+    id: '3'
+    taskid: f44d74e3-70f4-4376-8da4-db69193bfc8e
+    type: title
+    task:
+      id: f44d74e3-70f4-4376-8da4-db69193bfc8e
+      version: -1
+      name: Test Success
+      type: title
+      iscommand: false
+      brand: ''
+      description: ''
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 1070\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '4':
+    id: '4'
+    taskid: 3351f678-f10e-4ff8-8c7b-6aa96a8fc503
+    type: regular
+    task:
+      id: 3351f678-f10e-4ff8-8c7b-6aa96a8fc503
+      version: -1
+      name: HelloWorldScript
+      description: Hello World Script
+      scriptName: HelloWorldScript
+      type: regular
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '6'
+    scriptarguments:
+      name:
+        simple: World
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 720\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '5':
+    id: '5'
+    taskid: 8ddcc0ba-a4a8-4041-8593-eac896ba9a67
+    type: regular
+    task:
+      id: 8ddcc0ba-a4a8-4041-8593-eac896ba9a67
+      version: -1
+      name: DeleteContext
+      description: Delete field from context
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '4'
+    scriptarguments:
+      all:
+        simple: 'yes'
+      index: {}
+      key: {}
+      keysToKeep: {}
+      subplaybook: {}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 545\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '6':
+    id: '6'
+    taskid: 67819189-10b9-4690-8966-7e33a39d4feb
+    type: condition
+    task:
+      id: 67819189-10b9-4690-8966-7e33a39d4feb
+      version: -1
+      name: Verify outputs
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      'yes':
+      - '3'
+    separatecontext: false
+    conditions:
+    - label: 'yes'
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: HelloWorld.hello
+            iscontext: true
+          right:
+            value:
+              simple: Hello World
+    view: "{\n  \"position\": {\n    \"x\": 50,\n    \"y\": 895\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+view: "{\n  \"linkLabelsPosition\": {},\n  \"paper\": {\n    \"dimensions\": {\n \
```

Comment:
this is a result of using `update_playbook.py`, is that valid in demisto?

Reply:
yes, afaik